.github/workflows/build_and_test.yml

```diff
@@ -448,6 +514,11 @@ jobs:
         cd tests
         pip install -r requirements.txt
         pytest --ignore=simulator/ --doctest-modules
+    - name: Upload package
+      if: github.event_name == 'push' && github.event.push.ref == 'refs/heads/develop' && needs.check_changes.outputs.tket_changed == 'true'
+      run: |
+        conan user -p ${{ secrets.JFROG_ARTIFACTORY_TOKEN_2 }} -r tket-libs ${{ secrets.JFROG_ARTIFACTORY_USER_2 }}
+        conan upload tket/${{ needs.check_changes.outputs.tket_ver }}@tket/stable --all -r=tket-libs
```

Comment:
This is missing the logout ( `-c` ) ?

Reply:
It's only necessary on the self-hosted M1 machine (I didn't want the login to persist there, but the cloud VMs just get torn down anyway).